### PR TITLE
Added install function to static-get

### PR DESCRIPTION
**Pull request by the-djdj**
Added the ability to tell static-get to install a downloaded and extracted package.

### CODE REVIEW

File: static-get [hunk 1]
```diff
@@ -369,7 +370,26 @@ _static_get() {
                 if [ "${_sget__hash}" = "${_sget__hash_new}" ]; then
                     _cat "${tmpcache}.${uniq_id_mirror}/${_sget__bfile}"
 
-                    if [ "${extract}" ]; then
+                    if [ "${install}" ]; then
+                        _mkdir_p "${directory}/${_sget__bfile%%.tar*}"
+                        (cd "${directory}/${_sget__bfile%%.tar*}" && \
+                        ${compress_bin} < "../${_sget__bfile}"  | tar xf -) && \
```
Comment (javier-lopez) [inline static-get]: Not really sure about the `&& \` at the end of this line, I think it's unclear at best

File: static-get [hunk 2]
```diff
@@ -369,7 +370,26 @@ _static_get() {
                 if [ "${_sget__hash}" = "${_sget__hash_new}" ]; then
                     _cat "${tmpcache}.${uniq_id_mirror}/${_sget__bfile}"
 
-                    if [ "${extract}" ]; then
+                    if [ "${install}" ]; then
+                        _mkdir_p "${directory}/${_sget__bfile%%.tar*}"
```
Comment (javier-lopez) [inline static-get]: This code seems similar to the one in extract, do you think is it worth to create a function in order to reuse code?

**Review by javier-lopez — CHANGES_REQUESTED** [review]
Thank you for your collaboration 🍻 , I've left a short code review, also, would you consider adding some test cases for the new functionality?, https://github.com/minos-org/minos-static/blob/master/tests/tests.db

Have a nice day!

**Comment by the-djdj** [comment]
The issues should be fixed now. I couldn't figure out how to write a test case that invoked superuser privileges, so the tests focus on lack of write access to root.

**Comment by max-baz** [comment]
Nice feature! It would have been nice if we could configure a prefix, i.e. instead of installing to `/bin/` to be able to install to `/opt/bin/`

**Comment by the-djdj** [comment]
I'll take a look at that, @maximbaz

**Pull request merged**